src/bittytax/conv/parsers/coinbasepro.py

```diff
@@ -77,17 +77,18 @@ def _parse_coinbase_pro_row(
             buy_asset=row_dict["amount/balance unit"],
             wallet=WALLET,
         )
-    elif row_dict["type"] == "match":
+    elif row_dict["type"] in ("conversion", "match"):
```

Comment:
What is the "conversion" type for? Do you have an example?

Reply:
They look like a buy order and the transactions looked the same as "match" so they were easily handed there.
[conversion_example.csv](https://github.com/BittyTax/BittyTax/files/13795909/conversion_example.csv)